### PR TITLE
Add cross build to travis

### PR DESCRIPTION
**Pull request by saschagrunert**
This adds cross compilation targets to travis to avoid issues when
building for different platforms.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,19 +1,28 @@
...
 go:
-  - 1.10.x
-  - 1.11.x
-  - tip
-  - master
-
-env:
-  global:
-    - BUILDTAGS="selinux"
+  - 1.12.x
```
Comment (saschagrunert) [inline .travis.yml]: Let's move forward with go 1.12

Comment (rhatdan) [inline .travis.yml]: Should we test on 1.13 and tip?

**Comment by rhatdan** [comment]
LGTM other then question on versions.

File: .travis.yml (hunk 1 above)
Comment (saschagrunert) [reply .travis.yml]: Okay, I add those versions as well.

**Comment by rhatdan** [comment]
@kolyshkin @mrunalp @giuseppe PTAL

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**